Gulpfile.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const gulp = require('gulp');
+const bump = require('gulp-bump');
+const concat = require('gulp-concat');
+const git = require('gulp-git');
+const insert = require('gulp-insert');
+const jscs = require('gulp-jscs');
+const mocha = require('gulp-mocha-phantomjs');
+const rename = require('gulp-rename');
+const uglify = require('gulp-uglify');
+const webdriver = require('gulp-webdriver');
+
+const pkg = require('./package.json');
+const now = new Date().getFullYear();
+const banner = `/**
+ * ${pkg.description}
+ * ${pkg.name} v${pkg.version}, ${pkg.homepage}
+ * @copyright 2010-${now} ${pkg.author.name}, ${pkg.author.url}
+ * @license ${pkg.license}
+ */
+`;
+
+gulp.task('lint', () => {
+    return gulp.src('lib/*.js')
+        .pipe(jscs())
+        .pipe(jscs.reporter());
+});
+
+gulp.task('concat:lib', () => {
+    return gulp.src([
+        // Module system.
+        'node_modules/ym/modules.js',
+
+        // Core.
+        'lib/init.js',
+        'lib/shower.js',
+        'lib/*/*.js',
+
+        // Plugins.
+        'node_modules/shower-*/shower-*.js',
+    ])
+    .pipe(concat('shower.js'))
+    .pipe(gulp.dest('.'));
+});
+
+gulp.task('concat:test', () => {
+    return gulp.src('tests/unit/test.*.js')
+        .pipe(concat('tests.js'))
+        .pipe(gulp.dest('tests/unit'));
+});
+
+gulp.task('minify', [ 'concat:lib' ], () => {
+    return gulp.src('shower.js')
+        .pipe(uglify({ mangle: false }))
+        .pipe(rename({ suffix: '.min' }))
+        .pipe(insert.prepend(banner))
+        .pipe(gulp.dest('.'));
+});
+
+gulp.task('bump', () => {
+    return gulp.src('package.json')
+        .pipe(bump())
+        .pipe(gulp.dest('.'))
+        .pipe(git.add())
+        .pipe(git.commit('Bump version number'));
+        .on('end', () => {
+            git.push('origin', 'master', error => {
```

Comment:
Another way is to include `gulp-git-push`, [more on this](https://medium.com/@ferreiratiago/its-been-a-while-since-we-started-to-move-our-build-system-from-grunt-to-gulp-7571a6a2991#.ix34hkjrv).